ci-operator/step-registry/hypershift/mce/power/conformance/hypershift-mce-power-conformance-workflow.yaml

```diff
@@ -0,0 +1,27 @@
+workflow:
+  as: hypershift-mce-power-conformance
+  documentation: |-
+    The HyperShift mce power conformance workflow executes tests against a new ephemeral Hypershift Agent cluster
+    created on Power Nodes to validate the cluster creation reaches completed state.
+  steps:
+    pre:
+      - ref: ingress-aws-nlb-manifest
+      - chain: ipi-aws-pre
+      - ref: hypershift-mce-install
+      - ref: hypershift-mce-power-create
+    test:
+      - chain: hypershift-mce-power-test
+    post:
+      - chain: hypershift-mce-power-dump
+      - ref: hypershift-mce-power-destroy
```

Comment:
Should this invoke a step to destroy the control plane on AWS? 
`hypershift-mce-aws-destroy`

It doesn't look like `hypershift-mce-power-destroy` will clean up the AWS part